ios/Classes/Events/AirshipPushReceivedEvent.swift

```diff
@@ -16,20 +16,7 @@ class AirshipPushReceivedEvent : AirshipEvent {
 
     var data: Any? {
         get {
-            var payload = ["payload": notificationContent.notificationInfo]
-            if let notification = notificationContent.notification {
-                var notificaitonPayload : [String:Any] = [:]
-                notificaitonPayload["alert"] = notificationContent.alertBody
-                notificaitonPayload["title"] = notificationContent.alertTitle
-                notificaitonPayload["notification_id"] = notification.request.identifier
-
-                var extras = notificationContent.notificationInfo
-                extras["_"] = nil
-                extras["aps"] = nil
-                notificaitonPayload["extras"] = extras
-                payload["notification"] = notificaitonPayload
-            }
-
```

Comment:
I have the same issue with react native. Some informations are lost with the new update. @rlepinski thinks that we need to keep the current push body to avoid breaking integrations. I don't think it's possible, mybe we just need to add document on that. 
I think also you need to exhibit the extras only not all the payload. Not sure about that.